recipe/meta.yaml

```
@@ -1,5 +1,14 @@
 {% set name = "amrex" %}
 {% set version = "23.10" %}
+{% set build = 2 %}
```

Since #8 was merged:
```suggestion
{% set build = 3 %}
```